rviz_rendering/test/rviz_rendering/objects/point_cloud_test.cpp

```diff
@@ -259,11 +258,7 @@ TEST_F(
   renderables = point_cloud->getRenderables();
   for (auto const & renderable : renderables) {
     size_t number_of_vertices_per_box {0};
-    if (glsl_version >= 150) {
-      number_of_vertices_per_box = 1;
-    } else if (glsl_version >= 120) {
-      number_of_vertices_per_box = 6 * 3 * 2;  // six sides with two triangles each
-    }
+    number_of_vertices_per_box = point_cloud->getVerticesPerPoint();
```

Comment:
Nit: you can combine this statement with the one above, and just do:

```
size_t number_of_vertices_per_box = point_cloud->getVerticesPerPoint();
```

Also, in the loop in line 251 to 254, I think you should also call `getVerticesPerPoint`, as that seems to be better than hard-coding 1.

Reply:
Addressed with 37d7bf1eeb3adeb0f67f9095021a057fc731380a